### PR TITLE
Updated ember cli with ember-cli-update

### PR DESCRIPTION
**Pull request by kiwiupover**
Updated ember-cli to 3.3
Updated mocha to 5.2
Updated the tests to use chai
Updated babel to + 6.6 
Updated travis-ci to run tests against node 8 and 10

### CODE REVIEW

File: tests/unit/lib/ssh-nodetest.js [hunk 1]
```diff
@@ -152,13 +158,10 @@ describe('ssh', function() {
 
     it('passes expected parameters to the ssh2', function() {
 
-      var promise = subject.upload(options);
+      let promise = subject.upload(options);
 
       return assert.isFulfilled(promise)
         .then(function() {
-          var expectedKey = filePattern+':'+revisionKey;
```
Comment (kiwiupover) [inline tests/unit/lib/ssh-nodetest.js:159]: These are not used it this test

**Review by kiwiupover — COMMENTED** [review]
Cheers

**Comment by green-arrow** [comment]
I haven't used Ember in quite a long time.

@jheth - would you mind taking a look at this. Feel free to approve and merge.

**Comment by jheth** [comment]
Thanks @kiwiupover! This looks great.

**Comment by jheth** [comment]
@green-arrow Do I have access to npm publish? Do you have a typical tag/push routine you do?

**Pull request merged**